tests/test_multipitch.py

```diff
@@ -0,0 +1,285 @@
+        np.array([102., 84., 108.]),
+        np.array([98.745824285950576, 108.])
+    ]
+    actual = mir_eval.multipitch.frequencies_to_logscale(frequencies)
+    assert __frequencies_equal(actual, expected)
+
+
+def test_logscale_to_single_octave():
+    logscale_frequencies = [
+        np.array([96.]),
+        np.array([]),
+        np.array([102., 84., 108.]),
+        np.array([98.745824285950576, 108.])
+    ]
+    expected = [
+        np.array([0.]),
+        np.array([]),
+        np.array([6., 0., 0.]),
+        np.array([2.745824285950576, 0.])
+    ]
+    actual = mir_eval.multipitch.logscale_to_single_octave(
+        logscale_frequencies)
+    assert __frequencies_equal(actual, expected)
+
+
+def test_compute_num_freqs():
+    frequencies = [
+        np.array([256.]),
+        np.array([]),
+        np.array([362.03867196751236, 128., 512.]),
+        np.array([300., 512.])
+    ]
+    expected = np.array([1, 0, 3, 2])
+    actual = mir_eval.multipitch.compute_num_freqs(frequencies)
+    assert np.allclose(actual, expected, atol=A_TOL)
+
+
+def test_compute_num_true_positives():
+    ref_freqs = [
+        np.array([96., 100.]),
+        np.array([]),
+        np.array([81.]),
+        np.array([102., 84., 108.]),
+        np.array([98.745824285950576, 108.])
+    ]
+    est_freqs = [
+        np.array([96.]),
+        np.array([]),
+        np.array([200., 82.]),
+        np.array([102., 84., 108.]),
+        np.array([99., 108.])
+    ]
+    expected = np.array([1, 0, 0, 3, 2])
+    actual = mir_eval.multipitch.compute_num_true_positives(
+        ref_freqs, est_freqs)
+    assert np.allclose(actual, expected, atol=A_TOL)
+
+
+def test_accuracy_metrics():
+    true_positives = np.array([1, 0, 0, 3, 2])
+    n_ref = np.array([2, 0, 1, 3, 2])
+    n_est = np.array([1, 0, 2, 3, 2])
+
+    expected_precision = 0.75
+    expected_recall = 0.75
+    expected_accuracy = 0.6
+
+    (actual_precision,
+     actual_recall,
+     actual_accuarcy) = mir_eval.multipitch.accuracy(
+         true_positives, n_ref, n_est)
+
+    assert np.allclose(actual_precision, expected_precision, atol=A_TOL)
+    assert np.allclose(actual_recall, expected_recall, atol=A_TOL)
+    assert np.allclose(actual_accuarcy, expected_accuracy, atol=A_TOL)
+
+
+def test_error_score_metrics():
+    true_positives = np.array([1, 0, 0, 3, 2])
+    n_ref = np.array([2, 0, 1, 3, 2])
+    n_est = np.array([1, 0, 2, 3, 2])
+
+    expected_esub = 0.125
+    expected_emiss = 0.125
+    expected_efa = 0.125
+    expected_etot = 0.375
+
+    (actual_esub,
+     actual_emiss,
+     actual_efa,
+     actual_etot) = mir_eval.multipitch.error_score(
+         true_positives, n_ref, n_est)
+
+    assert np.allclose(actual_esub, expected_esub, atol=A_TOL)
+    assert np.allclose(actual_emiss, expected_emiss, atol=A_TOL)
+    assert np.allclose(actual_efa, expected_efa, atol=A_TOL)
+    assert np.allclose(actual_etot, expected_etot, atol=A_TOL)
+
+
+def unit_test_evaluate():
```

Comment:
Usually what we do is unit test the individual metrics instead of `evaluate`, just because then we know each of the individual metrics, when called, will raise the same errors (or different ones if individual metrics raise their own errors).